src/main/java/org/jabref/gui/exporter/ExportAction.java

```diff
@@ -152,7 +155,7 @@ public void update() {
     }
 
     private static FileDialogConfiguration createExportFileChooser(ExporterFactory exportFactory, String currentDir) {
-        List<FileType> fileTypes = exportFactory.getExporters().stream().map(Exporter::getFileType).collect(Collectors.toList());
+        Map<String, FileType> fileTypes = exportFactory.getExporters().stream().collect(Collectors.toMap(Exporter::getDisplayName, Exporter::getFileType));
```

Comment:
The logic to convert an exporter (and importer below) to a FileExtensionFilter should reside in `FileFilterConverter`.

Reply:
No, in this case it would not have any additional benefit. It would still return a map. The map is internally handled in the file dialog configuration and there handled.